readme.md

```diff
@@ -18,52 +18,174 @@ For API reference, please see the
 [H3 Documentation](https://h3geo.org/).
 
 **NOTE: This version (v.3.6.1) corresponds to the new Cython
-bindings, which will replace the old bindings (v3.4.3).
-This version is not yet published to PyPI.**
+bindings, which have replaced the old bindings (v3.4.3).**
 
 
 ## Install from PyPI
 
-TODO: instructions for installing from pre-built wheels
+`pip install h3`
 
+**New in v3.6.1**: We upload pre-built
+[Python Wheels to PyPI](https://pypi.org/project/h3) for Linux/Mac/Windows,
+which should avoid many previous installation issues.
 
-## Install from GitHub
+Try the library with:
 
-You need to have `cc`, `make`, `cmake`, and `git` in your `$PATH` when installing this package.
+```python
+>>> import h3
+>>> lat, lng = 0, 0
+>>> resolution = 0
+>>> h3.geo_to_h3(lat, lng, resolution)
+'8075fffffffffff'
+```
 
-Then run:
+## Example gallery
 
-`pip install git+https://github.com/uber/h3-py.git`
+Browse [a collection of example notebooks](https://github.com/uber/h3-py-notebooks),
```

Comment:
Are the notebooks here compatible with the latest bindings?

Reply:
I haven't tried, but they should be. These bindings are passing the same tests as the old bindings.

Reply:
I'll give them a whirl.

Reply:
The basic `Usage.ipynb` works, but the others need to be upgraded for Python 3. I'll make an issue about it for that repo.

Reply:
Starting a PR: https://github.com/uber/h3-py-notebooks/pull/2